qiskit_nature/problems/sampling/protein_folding/peptide/beads/main_bead.py

```diff
@@ -0,0 +1,77 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""A class defining a main bead of a peptide."""
+from typing import List
+
+from qiskit.opflow import PauliOp
+
+from ...peptide.beads.base_bead import BaseBead
+from ...peptide.chains.side_chain import SideChain
```

Comment:
```suggestion
from .base_bead import BaseBead
from ..chains.side_chain import SideChain
```
Maybe you can check the others in general, too..